code/__DEFINES/text.dm

```diff
@@ -7,3 +7,6 @@
 #define CENTER(text) {"<center>[##text]</center>"}
 
 #define SANITIZE_FILENAME(text) (GLOB.filename_forbidden_chars.Replace(text, ""))
+
+// for things that could be a list or a str, if its a list return rand element  - candy/ether
+#define PICK_ONE(x) islist(x) ? pick(x) : x
```

Comment:
Pick actually works on parameters as well as lists, meaning pick("hello") will always return hello, removing the need for this.

Reply:
..ah ok i overcompensated after assigning a list to say_verb by accident